src/components/buttons/facebook.tsx

```diff
@@ -19,6 +19,7 @@ export default styled(FacebookButton)`
   flex-direction: row;
 
   &:hover:not(:disabled) {
+    cursor: pointer;
```

Comment:
You only need to put it in the Base Button as all the other Button components extend from it.